packages/frontend/editor-ui/src/components/RunData.vue

```diff
@@ -810,7 +814,100 @@ function getNodeHints(): NodeHint[] {
 
 	return [];
 }
+function getConsolidatedHint(): { message: string } | null {
+	if (node.value?.disabled) {
+		return {
+			message: i18n.baseText('ndv.nodeHints.disabled'),
+		};
+	}
+	if (
+		canPinData.value &&
+		pinnedData.hasData.value &&
+		!editMode.value?.enabled &&
+		!props.isProductionExecutionPreview
+	) {
+		return null;
+	}
+	const activeSettings = [];
+
+	if (node.value?.alwaysOutputData) {
+		activeSettings.push({
+			key: 'alwaysOutputData',
+			message: i18n.baseText('ndv.nodeHints.alwaysOutputData'),
+			shortMessage: i18n.baseText('ndv.nodeHints.alwaysOutputData.short'),
+		});
+	}
+
+	if (node.value?.executeOnce) {
+		activeSettings.push({
+			key: 'executeOnce',
+			message: i18n.baseText('ndv.nodeHints.executeOnce'),
+			shortMessage: i18n.baseText('ndv.nodeHints.executeOnce.short'),
+		});
+	}
+
+	if (node.value?.retryOnFail) {
+		activeSettings.push({
+			key: 'retryOnFail',
+			message: i18n.baseText('ndv.nodeHints.retryOnFail'),
+			shortMessage: i18n.baseText('ndv.nodeHints.retryOnFail.short'),
+		});
+	}
+
+	if (
+		node.value?.onError === 'continueRegularOutput' ||
+		node.value?.onError === 'continueErrorOutput'
+	) {
+		activeSettings.push({
+			key: 'continueOnError',
+			message: i18n.baseText('ndv.nodeHints.continueOnError'),
+			shortMessage: i18n.baseText('ndv.nodeHints.continueOnError.short'),
+		});
+	}
+
+	if (activeSettings.length === 0) {
+		return null;
+	}
+
+	if (activeSettings.length === 1) {
```

Comment:
Would it be easier to always return an array of messages instead? And in template check if it's empty